controllers/utils/constants.go

```diff
@@ -41,7 +41,9 @@ const (
 
 // CR name length limits and suffix annotation
 const (
-	MaxPolicyNameLength    = 63
+	// Maximum length of policy + namespace (not including extra separator dot so 63 -1 = 62)
+	// this is calculated with utf8.RuneCountInString(policy.Name)+utf8.RuneCountInString+utf8.RuneCountInString(policy.Namespace)
+	MaxPolicyNameLength    = 62
```

Comment:
I prefer keep the same value here and count the dot in the length calculation. Or you want to rename it to maxPolicyNameLengthExcludingTheDot?

Reply:
fixed